scripts/sacred-worker

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python
+import sys
+import sacred.exp_runner
+sys.exit(sacred.exp_runner.main_worker())
```

Comment:
Why this extra file? Why not put the code from `exp_runner` here directly?

Reply:
I followed hyperopt's way of organizing here so I'm not strongly against moving exp_runner here. However I could argue that exp_runner is part of the main `sacred/` dir codebase, whilst the `script/` dir is only for executables/entrypoints.